applications/tari_walletd/web_ui/src/components/Memo.tsx

```
@@ -0,0 +1,22 @@
+//   Copyright 2025 The Tari Project
+//   SPDX-License-Identifier: BSD-3-Clause
+import { Memo } from "@tari-project/typescript-bindings";
+
+export type MemoProps = {
+  memo?: Memo | null;
+};
+
+export function Memo({ memo }: MemoProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Memo'. Consider to delete it or rename it.

'Memo' is defined here:


(lint/suspicious/noRedeclare)